lib/instantiate.js

```diff
@@ -72,10 +51,17 @@ define(function() {
 	 */
 	function isConstructor(func) {
 		var is = false, p;
-		for (p in func.prototype) {
-			if (p !== undef) {
-				is = true;
-				break;
+
+		// this has to work, according to spec:
+		// https://tc39.github.io/ecma262/#sec-function.prototype.tostring
+		is = is || func.toString().trim().substr(0,5) === 'class';
```

Comment:
Yep, looks right.  Nice find.  Any idea of the performance characteristics of this?  I figure it's probably fine, but just want to make sure we're not introducing a big perf hit by `toString`ing every `create`d function.